src/screens/main/index.js

```diff
@@ -0,0 +1,112 @@
+import React, {useState} from 'react';
+import {View, Text, TextInput, Alert} from 'react-native';
+import NUmericButtons from '../../components/NumericButtons';
+import OpButtons from '../../components/OpButtons';
+import {styles} from './index.style';
```

Comment:
```suggestion
import {styles} from './styles';
```
deberia llamarse styles nomas